services/slam/builtin/orbslam_int_test.go

```diff
@@ -66,35 +68,51 @@ func releaseImages(t *testing.T, mode slam.Mode) {
 	}
 }
 
-// Checks that we can get position and map, and that there are more than zero map points.
-// Doesn't check precise values due to variations in orbslam results.
-func testOrbslamPositionAndMap(t *testing.T, svc slam.Service) {
-	t.Helper()
-
-	position, err := svc.Position(context.Background(), "test", map[string]interface{}{})
-	test.That(t, err, test.ShouldBeNil)
-	// Typical values for RGBD are around (-0.001, -0.004, -0.008)
-	// Typical values for Mono without an existing map are around (0.020, -0.032, -0.053)
-	// Typical values for Mono with an existing map are around (0.023, -0.036, -0.040)
-	t.Logf("Position point: (%v, %v, %v)",
-		position.Pose().Point().X, position.Pose().Point().Y, position.Pose().Point().Z)
-	// Typical values for RGBD are around (0.602, -0.772, -0.202), theta=0.002
-	// Typical values for Mono without an existing map are around (0.144, 0.980, -0.137), theta=0.104
-	// Typical values for Mono with an existing map are around ( 0.092, 0.993, -0.068), theta=0.099
-	t.Logf("Position orientation: RX: %v, RY: %v, RZ: %v, Theta: %v",
-		position.Pose().Orientation().AxisAngles().RX,
-		position.Pose().Orientation().AxisAngles().RY,
-		position.Pose().Orientation().AxisAngles().RZ,
-		position.Pose().Orientation().AxisAngles().Theta)
+// Checks the orbslam map and confirms there are more than zero map points.
+func testOrbslamMap(t *testing.T, svc slam.Service) {
 	actualMIME, _, pointcloud, err := svc.GetMap(context.Background(), "test", "pointcloud/pcd", nil, false, map[string]interface{}{})
 	test.That(t, err, test.ShouldBeNil)
 	test.That(t, actualMIME, test.ShouldResemble, "pointcloud/pcd")
-	// Typical value for RGBD is 329
-	// Values for Mono vary
 	t.Logf("Pointcloud points: %v", pointcloud.Size())
 	test.That(t, pointcloud.Size(), test.ShouldBeGreaterThan, 0)
 }
 
+// Checks the orbslam position within a defined tolerance
+func testOrbslamPosition(t *testing.T, svc slam.Service, mode, actionMode string) {
+	var expectedPos r3.Vector
+	expectedOri := &spatialmath.OrientationVector{}
+	tolerancePos := 0.05
+	toleranceOri := 0.5
+
+	switch {
+	case mode == "mono" && actionMode == "mapping":
+		expectedPos = r3.Vector{X: 0.020, Y: -0.032, Z: -0.053}
```

Comment:
Defined expected position and orientation here as opposed to passing them into the function as it centralizes any changes to the expected values to one location.

Let me know if you'd prefer we handle this a different way.